examples/hpo/mlp.py

```diff
@@ -0,0 +1,235 @@
+import time
```

Comment:
Can you add comment at the top of this file providing a brief overview of what's being demonstrated?